fastlane/Fastfile

```diff
@@ -128,6 +128,12 @@ lane :tag_current_branch do |options|
   push_git_tags(tag: version_number)
 end
 
+desc "Update hybrid common pod and gradle"
+lane :update_hybrid_common_versions_and_open_pr do |options|
+  options[:open_pr] = true
```

Comment:
I wanted to reuse the code for local and automatic updates. Local updates are most likely run while preparing a local release, so we don't really want to open a PR in those cases